docs/_blog/2022-04-12-kuma-1-6-0.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Kuma 1.6.0 GA Released With Kubernetes Gateway API support, ZoneEgress improvements, a new and improved transparent-proxy and more.
+description: Kuma 1.6.0 GA Released With Kubernetes Gateway API support, ZoneEgress improvements, a new and improved transparent-proxy and more.
+date: 2022-04-12
+headerImage: /images/blog/kuma_1_6_blog_banner.jpg
+tags:
+  - Release
+canonicalUrl: 'https://kuma.io/blog/kuma-1-6-0-released'
+---
+
+We are happy to announce Kuma's latest release, which is packed with features and improvements, including substantial performance improvements to our built gateway and .
+
+We strongly suggest upgrading, in order to take advantage of the latest and greatest when it comes to service mesh.
+
+## Notable Features
+
+* 🚀 We provide a preview of [Kubernetes Gateway API](https://gateway-api.sigs.k8s.io/) support for our builtin gateway. This makes it easier than to provide a gateway to lead traffic through your mesh
+* 🚀 Full support for the "inspect API" on builtin gateway resources. This enables users to see which policies impact which `MeshGatewayRoute`.
```

Comment:
This sounds more general and makes it less noisier and clearer to understand for anyone not yet aware of the `MeshGatewayRoute` resource
```suggestion
* 🚀 Full support for the "inspect API" on builtin gateway resources. This enables users to see which policies impact which gateway routes.
```